app/controllers/concerns/filter_by_current_user_roles.rb

```diff
@@ -5,6 +5,8 @@ module FilterByCurrentUserRoles
     before_action :add_roles_to_filter_params!, only: :index
   end
 
+  # Filter controlled_resources by what role you have onto them,
```

Comment:
I'm inclined to leave it, I don't have the aversion to comments that the rest of the crew seems to feel. @marten wrote this one, anyway. :D